pkg/vpcmodel/explainabilityConnectivity.go

```diff
@@ -456,15 +472,15 @@ func (details *rulesAndConnDetails) computeConnections(c *VPCConfig,
 // if src or dst is a node then the node is from getCidrExternalNodes,
 // thus there is a node in VPCConfig that either equal to or contains it.
 func (v *VPCConnectivity) getConnection(c *VPCConfig, src, dst Node) (conn *detailedConn, err error) {
-	srcForConnection, err1 := c.getContainingConfigNode(src)
+	srcForConnection, err1 := c.getConnectedResource(src)
```

Comment:
A pip is translated/abstracted to its LB?

Reply:
yes